nestkernel/archiving_node.cpp

```diff
@@ -214,6 +214,9 @@ nest::Archiving_Node::get_status( DictionaryDatum& d ) const
   def< double >( d, names::tau_Ca, tau_Ca_ );
   def< double >( d, names::beta_Ca, beta_Ca_ );
   def< double >( d, names::tau_minus_triplet, tau_minus_triplet_ );
+  // By default, the neuron does't support the precise integration scheme.
+  // This property can be only set by the neuron implementation and not from outside.
+  def< bool >(d, names::is_precise_neuron, this->is_off_grid());
```

Comment:
I would move this all the way up to the `Node` base class. There is no principle reason why one would not derive a precise spiking subclass directly from `Node`.

Also, we don't need `this->` here.